src/components/Headers/DefaultHeader/DefaultHeader.tsx

```diff
@@ -8,13 +8,20 @@ import AddIcon from '@/assets/icons/AddIcon'
 import Link from 'next/link'
 import { useSession, signIn } from 'next-auth/react'
 import { useRouter } from 'next/navigation'
+import { createInitialPartOfUser } from '@/lib/createInitialPartOfUser'
 
 const DefaultHeader = () => {
   const { data: session, status } = useSession()
-  console.log(session)
   const router = useRouter()
 
   if (status === 'authenticated') {
+    const user = {
+      email: session.user.email || '',
+      image: session.user.image || '',
+      name: session.user.name || '',
+    }
```

Comment:
**issue:** Creating user should not work like that, it should be created on the backend side in authorization logic, not this way in UI